modules/op-tkn-clustertask-management.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-clustertask-management-commands_{context}"]
+= ClusterTask management commands
+
+== clustertask
+Manage ClusterTasks.
```

Comment:
Some of these types are written with space and some without space, cluster task use both styles ;)

Reply:
Hi @ppitonak , I did a ctrl+f for "cluster task", however I am unable to find any reference. Can you please point me to the specific instance where you see it.